ntbs-service/Pages/Notifications/Edit/_ClinicalDetailsDates.cshtml

```diff
@@ -9,18 +9,17 @@
 
 <h2>Clinical Dates</h2>
 
-<date-comparison inline-template>
+<date-comparison inline-template v-bind:number-of-dates="6">
     <div>
-        <div v-if="filteredDateWarnings.length > 0">
-            <govuk-warning-text id="nhs-number-warning" ref="nhs-number-warning">
+        <div class="hidden" ref="date-warning-0">
```

Comment:
I might have been inclined to stash the current rank/index in a variable and string-interpolate it everywhere...
Very happy to be ignored on this one - because I'm not sure I even like it - it would mean that changes down the line to this area would be more difficult to introduce breaking changes though (I think).